journal/modernized/32-epistles.adoc

```diff
@@ -218,17 +225,21 @@ or encumbered with the things of this life; but be ready for sufferings,
 which may attend that nation before many be aware, that we may all be ready,
 as Christ`'s freemen, to drink that cup which the Lord is pleased to put into our hands,
 for the trial of our faith, which is more precious than gold.
+
+[.signed-section-closing]
 My dear and true love is with you all, in the power of an endless life,
 wherein I am Your friend and brother,
```

Comment:
What Jared is suggesting here is that... sometimes it looks nice to bump down part of the last line of a letter (if there is a clear place to do it) and label it as a [.signed-section-closing].  So here it would be:

```ASCIIDOC
My dear and true love is with you all, in the power of an endless life,
wherein I am, 

[.signed-section-closing]
Your friend and brother,

[.signed-section-signature]
William Edmundson
```